src/synthesiser/Synthesiser.cpp

```diff
@@ -1961,7 +1961,7 @@ void Synthesiser::emitCode(std::ostream& out, const Statement& stmt) {
 #define UNARY_OP(opcode, ty, op)                \
     case FunctorOp::opcode: {                   \
         out << "(" #op "(ramBitCast<" #ty ">("; \
-        visit(*args[0], out);                    \
+        dispatch(*args[0], out);                    \
```

Comment:
This is picky but can you align the backslash. @mmcgr would insist on this.

Reply:
Done